### PR TITLE
api key changes

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

**Comment by puja108** [comment]
RFR @marians @luebken

File: src/main/java/currentweather/App.java [hunk 1]
```diff
@@ -44,8 +44,10 @@ private static String getCurrentWeather() throws IOException, JSONException {
 
       // if weather was not cached then make an API call and cache the result
       if (result == null) {
+      	String OpenWeatherMapAPIKey = "182564eaf55f709a58a13c40086fb5bb";
```
Comment (luebken) [inline src/main/java/currentweather/App.java]: local variable start with a small letter. but maybe its better to have a constant?

Comment (puja108) [inline src/main/java/currentweather/App.java]: Yes, I'll try with a constant, need to see what the best-practice for constants in Java is, but I have a clue.

Comment (puja108) [inline src/main/java/currentweather/App.java]: done

**Comment by marians** [comment]
LGTM

**Comment by marians** [comment]
Taking liberty to merge and deploy

**Pull request merged**